icu4j/main/tests/core/src/com/ibm/icu/dev/test/timezone/TimeZoneOffsetLocalTest.java

```diff
@@ -226,11 +227,11 @@ public void TestGetOffsetAroundTransition() {
             }
         }
 
-        // Test getOffsetFromLocal(long time, int nonExistingTimeOpt, int duplicatedTimeOpt, int[] offsets)
-        // with nonExistingTimeOpt = LOCAL_LATTER/duplicatedTimeOpt = LOCAL_FORMER
+        // Test getOffsetFromLocal(long time, LocalOption nonExistingTimeOpt, LocalOption duplicatedTimeOpt, int[] offsets)
+        // with nonExistingTimeOpt = LATTER/duplicatedTimeOpt = FORMER
         for (int i = 0; i < TESTZONES.length; i++) {
             for (int m = 0; m < MILLIS.length; m++) {
-                TESTZONES[i].getOffsetFromLocal(MILLIS[m], BasicTimeZone.LOCAL_LATTER, BasicTimeZone.LOCAL_FORMER, offsets);
+                TESTZONES[i].getOffsetFromLocal(MILLIS[m], LocalOption.LATTER, LocalOption.FORMER, offsets);
                 if (offsets[0] != OFFSETS3[m][0] || offsets[1] != OFFSETS3[m][1]) {
                     errln("Bad offset returned by " + TESTZONES[i].getID() + " at "
                             + df.format(new Date(MILLIS[m])) + "(wall/LATTER/FORMER) - Got: "
```

Comment:
Could we extend the test to test the cases for 
5) nonExistingTimeOpt = LocalOption.STANDARD_LATER , duplicatedTimeOpt = LocalOption.STANDARD_FORMER
6) nonExistingTimeOpt = LocalOption.DAYLIGHT_FORMER , duplicatedTimeOpt =LocalOption. DAYLIGHT_LATTER  
and ensure the return are correct.

Reply:
Same as above.

Reply:
could you file a ticket and add a TODO with that ticket number here at least. thanks